pkg/version/versionutil/versionutil.go

```diff
@@ -38,3 +38,9 @@ func GreaterThan(limaVersion, oldVersion string) bool {
 	}
 	panic("unreachable")
 }
+
+// GreaterEqual return true if limaVersion >= oldVersion.
+func GreaterEqual(limaVersion, oldVersion string) bool {
+	return strings.TrimPrefix(limaVersion, "v") == strings.TrimPrefix(oldVersion, "v") ||
+		GreaterThan(limaVersion, oldVersion)
+}
```

Comment:
This function is subtly different from `GreaterThan` because it doesn't strip the git describe suffix from `limaVersion`.

I think it would be better to use something like this (untested):
```go
func compare(limaVersion, oldVersion string) int {
	if limaVersion == "" {
		return -1
	}
	version, err := Parse(limaVersion)
	if err != nil {
		return 1
	}
	cmp := version.Compare(*semver.New(oldVersion)) {
    if cmp == 0 && strings.Contains(limaVersion, "-") {
        cmp = 1
    }
    return cmp
}

func GreaterThan(limaVersion, oldVersion string) bool {
    return compare(limaVersion, oldVersion) > 0
}

func GreaterEqual(limaVersion, oldVersion string) bool {
    return compare(limaVersion, oldVersion) >= 0
}
```

Reply:
Also please add tests for `GreaterEqual`.

Reply:
done